### PR TITLE
Document timestamp

### LINKED ISSUE #2111: block_timestamp doesn't document units
```
    /// The current block timestamp.
    pub block_timestamp: u64,
```

```
/// Current block timestamp.
pub fn block_timestamp() -> u64 {
```

Note that the common issue is that generally comments which are like `/// Method name` are completely useless.
Comments should provide info which isn't easily inferred from method name, e.g. in this case which units is timestamp measured in and reference point. Without this info one doesn't have any idea how to use this field.

### PR DESCRIPTION
**Pull request by bowenwang1996**
Document the unit and reference point of timestamp. Fixes #2111. Also update the docstring in near-bindgen https://github.com/nearprotocol/near-bindgen/pull/86

### CODE REVIEW

**Comment by vgrichina** [comment]
Needs to update https://github.com/nearprotocol/nearcore/blob/d68cad0a8977debeba543ad90ad3a08721188481/runtime/near-vm-logic/src/logic.rs#L514 as well I think

also pretty sure there are occurrences in near-bindgen, e.g. `src/environment/env.rs`

File: runtime/near-vm-logic/src/context.rs [hunk 1]
```diff
@@ -24,7 +24,7 @@ pub struct VMContext {
     /// The current block height.
     // TODO #1903 rename to `block_height`
     pub block_index: BlockHeight,
-    /// The current block timestamp.
+    /// The current block timestamp (number of non-leap-nanoseconds since January 1, 1970 UTC).
```
Comment (SkidanovAlex) [inline runtime/near-vm-logic/src/context.rs]: Since what exact moment on January 1st 1970?

**Comment by vgrichina** [comment]
@bowenwang1996 @SkidanovAlex @nearmax any reason we need 6+ days to merge tiny doc change?

**Comment by bowenwang1996** [comment]
@evgenykuzyakov's approval is needed and wasn't there until yesterday.

**Comment by fckt** [comment]
Minor thing, but still: is there any reason for such a precision (nanoseconds) for a blocktime?

**Comment by lazaridiscom** [comment]
your @codecov is a bit too verbose, highly annoying, especially within doc changes.

**Comment by bowenwang1996** [comment]
@fckt is there any reason not to? If people don't want to use such precision they can round up, but if we don't provide precision up to nanosecond for people who actually want such precision they would have no way of doing so.

**Pull request merged**